package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "bb-practicum-team-1-back",
+  "name": "Beauty Finder GG-PRAC-TEAM5",
```

Comment:
I don't know if that matters, yet according to CTD's pattern this should be
gg-practicum-team-5-back. I leave it up to you

Reply:
Done